gradio/chat_interface.py

```diff
@@ -203,108 +206,111 @@ def __init__(
                     break
 
         with self:
-            if title:
-                Markdown(
-                    f"<h1 style='text-align: center; margin-bottom: 1rem'>{self.title}</h1>"
-                )
-            if description:
-                Markdown(description)
-            if chatbot:
-                if self.type and self.type != chatbot.type:
-                    warnings.warn(
-                        "The type of the chatbot does not match the type of the chat interface. The type of the chat interface will be used."
-                        "Recieved type of chatbot: {chatbot.type}, type of chat interface: {self.type}"
+            with Column():
```

Comment:
This `Column()` is needed b/c otherwise if you put `gr.ChatInterface` in a `gr.Row()`, the textbox appears to the right of the chatbot instead of below it.